src/tim/vx/operation.cc

```diff
@@ -43,6 +43,7 @@ OperationImpl::OperationImpl(Graph* graph, uint32_t operation_id,
                            output_cnt_, NULL)) {
   SetRoundingPolicy();
   node_->uid = graph_->graph()->cur_nid;
+  node_->attr.enable_op_constraint_check = false;
```

Comment:
Why do we disable constraint check? If we want to do this, let's move this into a separate patch.

Reply:
got it! I've remove it from this patch.